core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/VnodeRepairGroupFactory.java

```diff
@@ -60,7 +62,7 @@ private VnodeRepairGroupFactory()
                         .map(VnodeRepairState::getTokenRange)
                         .collect(Collectors.toList());
```

Comment:
There is a "nice" Guava-feature (but not added until Guava 21) so that you can make the stream return an Immutable collection directly:

.collect(ImmutableList.toImmutableList());

-> no need to call the ImmutableList.copyOf()

But that is maybe for another commit :-)